heka/_service.sls

```diff
@@ -112,14 +112,24 @@ heka_{{ service_name }}_service:
     'splitter': {},
     'encoder': {},
     'output': {},
+  },
+  'os_telemetry_collector': {
+    'decoder': {},
+    'input': {},
+    'trigger': {},
+    'alarm_cluster': {},
```

Comment:
You can remove `trigger` and `alarm_cluster`, as no alarms will be configured in the telemetry collector.